test/tableau/extensions/sitemap_extension_test.exs

```diff
@@ -0,0 +1,75 @@
+defmodule Tableau.SitemapExtensionTest do
+  use ExUnit.Case, async: true
+
+  alias Tableau.SitemapExtension
+
+  test "use _site as default out dir" do
+    token = %{
+      site: %{
+        config: %{
+          url: "http://example.com"
+        },
+        pages: [
+          %{
+            date: ~U[2018-02-28 00:00:00Z],
+            permalink: "/about",
+            title: "About"
+          }
+        ]
+      }
+    }
+
+    SitemapExtension.run(token)
+
+    sitemap = File.read!("_site/sitemap.xml")
+
+    assert sitemap ===
+             :erlang.iolist_to_binary([
```

Comment:
you can use `IO.iodata_to_binary()`

Reply:
You can probably just make it a heredoc tho

Reply:
Thanks, I changed it to heredoc